client/js/components/statement/publicStatementLists/DpPublicStatementList.vue

```diff
@@ -261,8 +261,8 @@ export default {
       const statementParagraph = (paragraph && paragraph.title) || Translator.trans('notspecified')
       const text = statement.text
 
-      const transformedSubmitDate = submitted === false ? {} : { submittedDate: dayjs(submittedDate).format('DD.MM.YYYY HH:mm') }
-      const transformedCreatedDate = dayjs(createdDate).format('DD.MM.YYYY HH:mm')
+      const transformedSubmitDate = submitted === false ? {} : { submittedDate: dayjs.unix(statement.submit).format('DD.MM.YYYY HH:mm') }
+      const transformedCreatedDate = dayjs.unix(statement.createdDate / 1000).format('DD.MM.YYYY HH:mm')
```

Comment:
What is achieved by this operation?

Reply:
Well the timestamp of `createdDate` and `submittedDate` are in different formats (one in miliseconds the other in seconds) so to display the correct date this operation was required, otherwise a statement contained a weird created date in the header)

Reply:
I dislike the idea of fixing inconsistent data in the frontend. 

> Well the timestamp of createdDate and submittedDate are in different formats (one in miliseconds the other in seconds) 

Are they persisted like that in the database? I do not think so... where does the bug originate from?

Reply:
Yes they are persisted like that in the database and DpPubblicStatementList.vue receives the statement data via the templateVars which will be transformed in the function above. Would your proposal be to tell the backend we are going to need consistent data?